kolibri/core/apps.py

```diff
@@ -84,3 +90,54 @@ def activate_pragmas_on_start():
             # at the cost of a slight penalty to all reads.
             cursor.execute(START_PRAGMAS)
             connection.close()
+
+    @staticmethod
+    def check_redis_settings():
+        """
+        Check that Redis settings are sensible, and use the lower level Redis client to make updates
+        if we are configured to do so, and if we should, otherwise make some logging noise.
+        """
+        from kolibri.core.utils.cache import process_cache
```

Comment:
Did this throw an app registry not ready error if imported outside of the scope here?

Reply:
No, I didn't run into that, but I also didn't try. It seemed perhaps a little safer overall, but I guess if it doesn't break with it outside, we'll be okay.